OAuth2/Client/Impl/TwitterClient.cs

```diff
@@ -109,14 +110,14 @@ protected override UserInfo ParseUserInfo(string content)
             return new UserInfo
             {
                 Id = response["id"].Value<string>(),
-                Email = null,
+                Email = response["email"] == null ? null : response["email"].Value<string>(),
```

Comment:
It would be better to call SafeGet to get the string value here so we don't blow up.